CenterForOpenScience/osf.io
# [Migration Required] Feature/Zotero Personal Library Data Migration [PLAT-496]

## Purpose

Data migration to make existing zotero addon configurations work. Previously, you only had the capability to fetch citations from personal libraries. https://github.com/CenterForOpenScience/osf.io/pull/7803 adds the ability to pull citations from group libraries.

## Changes

Adds migration to previous zotero node settings and user settings to mark there existing library_ids as "personal" because those are the only libraries that exist.
## QA Notes
 Same as existing test sessions on PLAT-496, but focusing on zotero addons that were configured before the migration.  Make sure "All documents" or subfolders are still connected.

Note: You will not be able to see new group libraries until you disconnect zotero and reauth in order to get a key with the proper permissions that will allow you to see groups.

## Side Effects



## Ticket

https://openscience.atlassian.net/browse/PLAT-496
